data/installed-tests/fwupdmgr-p2p.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# ---
+echo "Starting P2P daemon..."
+FWUPD_DBUS_SOCKET=/var/run/fwupd.sock /usr/libexec/fwupd/fwupd --verbose --timed-exit --no-timestamp &
```

Comment:
for the installed test, don't you want to use a systemd unit with the socket defined and managed to start it?

Reply:
Is systemd running in the CI?

Reply:
At least for Debian it seems to be:

```
        service dbus restart
        gnome-desktop-testing-runner fwupd
        apt purge -y fwupd-tests
```

Reply:
Doesn't systemd allow environment override files? My idea was create one that sets dbus variable.

Although for installed tests on real systems this falls apart.
So scratch this idea.

However I think you want to make sure the socket exists rather than just sleep to make this more foolproof.